### ISSUE TITLE
Pixmap resizing needs positional arg "clip" - even if None

### ISSUE THREAD

**Issue by paaguti** [pymupdf/PyMuPDF#3139]
### Description of the bug

I'm trying to resize a pixmap with
```
    npixmap = fitz.Pixmap(pixmap, new_width, new_height)
```

And that throws the following exception

```
Traceback (most recent call last):
  File "/home/paag/bin/img2pdf", line 129, in <module>
    main()
  File "/home/paag/bin/img2pdf", line 89, in main
    npixmap = fitz.Pixmap(pixmap, new_width, new_height)
  File "/home/paag/.local/lib/python3.10/site-packages/fitz/__init__.py", line 9471, in __init__
    elif args_match(args, (Pixmap, mupdf.FzPixmap), (int, None)):
  File "/home/paag/.local/lib/python3.10/site-packages/fitz/__init__.py", line 17869, in args_match
    if type_ is not None and not isinstance(args[j], type_):
TypeError: isinstance() arg 2 must be a type, a tuple of types, or a union
```

### How to reproduce the bug

This is the MWE to reproduce the Exception:

```
XINCH=8.268
YINCH=11.693
dpi=50
f = 'test.png'
pixmap = fitz.Pixmap(f)
ratio = pixmap.height/pixmap.width
new_width = XINCH * dpi
new_height = new_width * ratio
npixmap = fitz.Pixmap(pixmap, new_width, new_height)
```
The intent is that npixmap can be included in a 50 dpi A4 pdf

### PyMuPDF version

1.23.21

### Operating system

Linux

### Python version

3.10

**Comment by JorjMcKie** [comment]
Given this, I will change the function to make the last parameter `None`/`clip` optional.
For now, your code should work, if you add `None` as the last parameter.

**Comment by paaguti** [comment]
Thx for the prompt answer

**Comment by julian-smith-artifex-com** [comment]
Fixed in 1.23.22.
